src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -99,11 +169,66 @@ const FaultEventMenu = ({ shapeToolData, onEventUpdated, refreshTree, rootIri }:
     } else {
       setSchematicDesignation(undefined);
     }
+
+    let supertypes: any = shapeToolData?.supertypes?.supertypes;
+
+    if (shapeToolData?.selectedEstimate) {
+      // SELECTED ESTIMATE => PREDICTED OR OPERATIONAL IS SELECTED
+      const iriOfSelectedValue = shapeToolData.selectedEstimate.iri;
+      const { predictionIri, operationalIri } = supertypes.reduce(
+        (acc, item) => {
+          if (item?.hasFailureRate?.prediction?.iri) acc.predictionIri = item.hasFailureRate.prediction.iri;
+          if (item?.hasFailureRate?.estimate?.iri) acc.operationalIri = item.hasFailureRate.estimate.iri;
+          return acc;
+        },
+        { predictionIri: "", operationalIri: "" },
+      );
+
+      if (iriOfSelectedValue === predictionIri) {
+        setSelectedRadioButton(RadioButtonType.Predicted);
+      } else if (iriOfSelectedValue === operationalIri) {
+        setSelectedRadioButton(RadioButtonType.Operational);
+      }
+      setSnsManuallyDefinedFailureRate(shapeToolData?.probability);
+    } else {
+      // NO SELECTED ESTIMATE => MANUAL IS SELECTED
+      setSelectedRadioButton(RadioButtonType.Manual);
+      if (shapeToolData?.probability) {
+        setSnsManuallyDefinedFailureRate(shapeToolData.probability);
+        setExternalManuallyDefinedFailureRate(shapeToolData.probability);
```

Comment:
I do not understand this ... why we have `setExternalManuallyDefinedFailureRate(shapeToolData.probability); ` in the `else` block if for external event predicted and operational failure rates do not exist.

Moreover, we do not have any basicManuallyDefinedFailureRate

Reply:
created a sep. issue for this one: https://github.com/kbss-cvut/fta-fmea-ui/issues/409